### ISSUE TITLE
Unblock when we can figure out how to check the existence of the image

### ISSUE THREAD

**Issue by aronchick** [bacalhau-project/bacalhau#843]
```
		// TODO: Unblock when we can figure out how to check the existence of the image
```

https://github.com/filecoin-project/bacalhau/blob/b3b548f9af4c78aba961bc72061fc63513c70bd9/cmd/bacalhau/utils_test.go#L136

**Comment by aronchick** [comment]
@rossjones is this possible now? Just curious

**Comment by rossjones** [comment]
tldr; No. It's not possible today.

Test was moved to https://github.com/bacalhau-project/bacalhau/blob/main/cmd/util/version_test.go#L128-L136 

This is client-side job validation, used to check that nothing is strucurally wrong with the job being submitted, but doesn't verify the contents of the job (e.g. it doesn't check http URLs you might provide to -i) and doesn't check with the docker hub whether the image exists.  

Arguably the `DockerEngineBuilder` should ensure that the image is non-blank (as a minimum) but really only the requester can currently check the validity (or not) of the docker image provided.

**Comment by aronchick** [comment]
K - sadly closing :(
